landsites_tools/interface_settings.py

```diff
@@ -109,6 +123,56 @@ def machine(self):
         """TO DO: machine description and implementation (if needed)"""
         return self.parser['user']['machine']
 
+    ############################################################################
+    '''
+    Model run parameters
+    '''
+
+    @property
+    def start_date(self):
+        """Sites to be simulated"""
+        return self.parser['model_params']['start_date']
+
+    @property
+    def end_date(self):
+        """Sites to be simulated"""
+        return self.parser['model_params']['end_date']
```

Comment:
I think it would be better to store dates as Python [datetime objects](https://docs.python.org/3/library/datetime.html) (using [the dateutil parser](https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.parse)) (as in https://github.com/NorESMhub/NorESM_LandSites_Platform/blob/platform_dev/LandsitesTools/interface_settings.py#L55), because we will likely need this information for other data structures in the platform than just setting simulation start and end dates.

Datetime objects can be converted to strings according to the format required by each application (https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior)

Reply:
Good points! Removed it because I thought we would only need it as an input parameter in a fixed format, but after reflecting on it a bit more, I completely agree with you